DevoxxClientMobile/src/main/java/com/devoxx/views/BadgesPresenter.java

```diff
@@ -125,9 +92,17 @@ private void loadAnonymousView() {
     }
 
     private void loadAuthenticatedView() {
+        badgesView.setCenter(content);
```

Comment:
Once the user has chosen either to be a sponsor or attendee, each consecutive time that the badges view is opened, I would not expect to ever need to make that choice again?
Actually, the only reason I can think of to keep this behaviour is that a sponsor could be an attendee and would want to be able to scan badges both as a sponsor and as an attendee?

Reply:
Yes, I was of the opinion that an user can be both attendee and sponsor. Although, I am not entirely sure of the use-case.